messages/config-writer.md

```diff
@@ -1,3 +1,7 @@
 # error.unrecognized-file-format
 
 The output file %s has an unsupported extension. Valid extensions include: .yaml/.yml.
+
+# prompt.overwrite-existing-file
+
+A file named %s already exists. Do you want to overwrite it?
```

Comment:
FYI @jshackell-sfdc

Reply:
I already ran this message by her, and this is the revised version.